plugins/node/opentelemetry-instrumentation-cassandra/src/instrumentation.ts

```diff
@@ -45,6 +45,14 @@ export class CassandraDriverInstrumentation extends InstrumentationBase {
     super('@opentelemetry/instrumentation-cassandra-driver', VERSION, config);
   }
 
+  private _getConfig(): CassandraDriverInstrumentationConfig {
+    return this._config;
+  }
+
+  override setConfig(config?: CassandraDriverInstrumentationConfig): void {
+    super.setConfig(config);
+  }
+
```

Comment:
Are these any different from the base implementation? There are `getConfig` and `setConfig` defined on the InstrumentationBase.

Reply:
Not really the only reason for these was the return/parameter types. Let me know what you think of [the changes](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1180/commits/f0e3bfca434ed26bd14f8f56a1870285a0cf7535) :)